### DISCUSSION TITLE
Feature Request: Ability to import transactions which are exported as xls from Value Research Online (VRO)

### DISCUSSION THREAD

**Discussion by LEvinson2504** [ananthakumaran/paisa#155] [Ideas]
I have written a simple template to import data from VRO excel quickly, we can add our previous transactions easily this way. I didn't see the option to import from VRO but there was an xls import which helped me, maybe we can add this as an option aswell. Happy to raise a PR for this if.

```
{{#if (isDate ROW.A "YYYY-MM-DD")}}
 {{date ROW.A "YYYY-MM-DD"}} {{ROW.C}}
    Assets:Equity:{{ROW.B}}		{{ROW.E}} {{acronym ROW.B}} @ {{ROW.F}} INR 
    Assets:Savings:                       {{negate ROW.H}} INR
    Expense:Misc   ;; balance any rounding errors
{{/if}}
```

**Comment by ananthakumaran** [comment]
You can share an anonymized version of the xls file here, I will include the template to Paisa.

**Reply by LEvinson2504** [reply]
It also has a PPF section we could add, but the date format is different, also included it in the file.

[dummy_vro_transactions.xls](https://github.com/ananthakumaran/paisa/files/14000923/dummy_vro_transactions.xls)

**Reply by ananthakumaran** [reply]
Thanks, added the template with some minor tweaks https://github.com/ananthakumaran/paisa/commit/507d230cca44c8edd850566655c3614757f0083f
